src/index/base.h

```diff
@@ -100,6 +100,8 @@ class BaseIndex : public CValidationInterface
     /// not block and immediately returns false.
     bool BlockUntilSyncedToCurrentChain();
 
+    bool IsSynced();
```

Comment:
8bb1ca8a276f5f6fc5af3639c2d58c496b71bf8f 

`bool IsSyned() const;`. Any reason to not write implementation here?

Reply:
Fixed.